esp/templates/users/userview.html

```diff
@@ -30,6 +30,9 @@ <h1>User Information</h1>
             {% endfor %}
         </select>
     </form>
+    {% if profile.teacher_info %}
```

Comment:
This doesn't catch my account, which is a teacher & student & admin. It catches people who are just teachers, but if we want to include this we should probably make it catch the case of teacher being just one of the roles?

Reply:
(same with line 97 though)

Reply:
Hmm...this is probably a broader issue related to how the profile works, but I should be able to fix this by just doing something like `{% if "Teacher" in user.getUserTypes %}`.

Reply:
(And then same for all the other user type conditionals)